db/schema.rb

```diff
@@ -267,6 +267,23 @@
     t.index ["team_id"], name: "index_conversations_on_team_id"
   end
 
+  create_table "csat_survey_responses", force: :cascade do |t|
+    t.bigint "account_id", null: false
+    t.bigint "conversation_id", null: false
+    t.bigint "message_id", null: false
+    t.integer "rating", null: false
+    t.text "feedback_text"
```

Comment:
feedback is enough right?

Reply:
At a glance, `csat_survey_responses.feedback` might be confusing, whether it's numeric feedback or text feedback.
We could also go with `additional_feedback` or  `feedback_message`.